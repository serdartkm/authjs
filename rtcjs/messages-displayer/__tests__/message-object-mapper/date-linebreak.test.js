import{h} from 'preact'
import MessageDateLine from '../../lib/message-object-mapper/date-linebreak'
import  render from 'preact-render-to-string';


describe("MessageDateLine component snapshot",()=>{
    it("renders correctly",()=>{
        const tree =render(<MessageDateLine datetime={1569393960355}/>)
         expect(tree).toMatchSnapshot();
    })
})
