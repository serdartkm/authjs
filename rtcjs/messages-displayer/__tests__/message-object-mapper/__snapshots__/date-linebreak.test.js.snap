// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MessageDateLine component snapshot renders correctly 1`] = `"<div style=\\"display: flex;\\"><div style=\\"flex: 1;\\"><hr /></div><div>9/25/2019</div><div style=\\"flex: 1;\\"><hr /></div></div>"`;
